server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -885,14 +885,6 @@ public enum Config {
             "0",
             "Default disk I/O writerate in bytes per second allowed in User vm's disk.",
             null),
-    ControlCidr(
-            "Advanced",
-            ManagementServer.class,
-            String.class,
-            "control.cidr",
-            "169.254.0.0/16",
-            "Changes the cidr for the control network traffic.  Defaults to using link local.  Must be unique within pods",
-            null),
     ControlGateway("Advanced", ManagementServer.class, String.class, "control.gateway", "169.254.0.1", "gateway for the control network traffic", null),
```

Comment:
@DaanHoogland 
should ControlGateway be dynamic as well ?

Reply:
I'll add it (and move it as well)